advanced_functionality/tensorflow_bring_your_own/container/cifar10/train

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+
+# A sample training component that trains a simple scikit-learn decision tree model.
+# This implementation works in File mode and makes no assumptions about the input file names.
+# Input is specified as CSV with a data point in each row and the labels in the first column.
+
+from __future__ import print_function
+
+import os
+import json
+import sys
+import subprocess
+import traceback
+
+# These are the paths to where SageMaker mounts interesting things in your container.
+prefix = '/opt/ml/'
+input_path = os.path.join(prefix,'input/data')
+output_path = os.path.join(prefix, 'output')
+model_path = os.path.join(prefix, 'model')
+param_path = os.path.join(prefix, 'input/config/hyperparameters.json')
+
+# This algorithm has a single channel of input data called 'training'. Since we run in
+# File mode, the input files are copied to the directory specified here.
+channel_name = 'training'
+training_path = os.path.join(input_path, channel_name)
+
+# default params
+training_script = 'cifar10.py'
+default_params = ['--model-dir', str(model_path)]
+
+
+# Execute your training algorithm.
+def _run(cmd):
+    process = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, env=os.environ)
+    stdout, stderr = process.communicate()
+
+    return_code = process.poll()
+    if return_code:
+        error_msg = 'Return Code: {}, CMD: {}, Err: {}'.format(return_code, cmd, stderr)
+        raise Exception(error_msg)
+
+
+def _hyperparameters_to_cmd_args(hyperparameters):
```

Comment:
You can do later: docstring explaining this function

Reply:
done